t/OptionFields.t

```diff
@@ -0,0 +1,133 @@
+# This file is part of Business:PayPal:API Module.   License: Same as Perl.  See its README for details.
+# -*- mode: cperl -*-
+use Test::More;
```

Comment:
The `strict` and `warnings` pragmas don't kick in until after they are used, so they should precede any other use statements.